project-koku/koku
# [COST-2439] aws legal entity column

Fixes [COST-2439](https://issues.redhat.com/browse/COST-2439)

Changes proposed in this PR:
* Adds the legal entity column as required for AWS
* Adds legal entity to daily group by columns
* Adds migration to drop aws daily Trino table (it will be recreated with the new column on next data load)

Testing Instructions:
1. Run standard test customer data flow on main
2. Switch to this branch
3. Run migrate_trino.py script to drop table
4. Run `show tables;` in trino to verify table is gone
5. Add and process a new source.
6. Check that table exists with new column. 

---
Additional Context:

[Smoke Tests](https://ci.ext.devshift.net/job/project-koku-koku-pr-check/4302/)
